### PR TITLE
Adding CRC 1.0.0 to the stacks

### PR DESCRIPTION
**Pull request by odockal**
- Adding CDK 3.10

Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

File: minishift.yaml [hunk 1]
```diff
@@ -833,3 +833,50 @@ availableRuntimes:
...
+   ########################
+   #       CRC 1.0.0      #
+   ########################
+
+ - &crc1000
+   id: crc1000
+   name: CRC v1.0.0
+   version: 1.0.0
+   license: *apache20
+   url: https://github.com/minishift/minishift/releases
```
Comment (jeffmaury) [inline minishift.yaml]: This URL needs to be updated

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: minishift.yaml (hunk 1 above)
Comment (odockal) [reply minishift.yaml]: true,true...

**Comment by odockal** [comment]
@jeffmaury I have also updated id to crc100 (was crc1000) to cover major.minor.release pattern. Please note that I am not sure about cdk 3.10 I have asked on the forum-crc if it is official or not.

**Comment by odockal** [comment]
@jeffmaury I have confirmed cdk 3.10 can be added as well.

**Review by jeffmaury — APPROVED** [review]

**Comment by odockal** [comment]
@jeffmaury I have checked that items are now available via download runtimes.

Thanks.

**Pull request merged**